cassandra/datastax/graph/graphson.py

```diff
@@ -141,7 +135,7 @@ def serialize(cls, value, writer=None):
 
     @classmethod
     def get_specialized_serializer(cls, value):
-        if type(value) in six.integer_types and (value > MAX_INT32 or value < MIN_INT32):
+        if type(value) in int and (value > MAX_INT32 or value < MIN_INT32):
```

Comment:
This should be the following instead:

```
if type(value) == int and (value > MAX_INT32 or value < MIN_INT32):
```

six.integer_types is just "int" in the Python3 case (it is a seq for Python2) so I'm not sure how this wasn't a problem before.